arkane/orca.py

```diff
@@ -0,0 +1,231 @@
+        Orca log file. If multiple such geometries are identified, only the
+        last is returned.
+        """
+        atoms, coords, numbers, mass = [], [], [], []
+
+        with open(self.path) as f:
+            log = f.readlines()
+
+        # First check that the Orca job file (not necessarily a geometry optimization)
+        # has successfully completed, if not an error is thrown
+        completed_job = False
+        for line in reversed(log):
+            if 'ORCA TERMINATED NORMALLY' in line:
+                logging.debug('Found a sucessfully completed Orca Job')
+                completed_job = True
+                break
+
+        if not completed_job:
+            raise LogError(
+                'Could not find a successfully completed Orca job in Orca output file {0}'.format(self.path))
+
+        # Now look for the geometry.
+        # Will return the final geometry in the file under Standard Nuclear Orientation.
+        geometry_flag = False
+        for i in reversed(range(len(log))):
+            line = log[i]
+            if 'CARTESIAN COORDINATES (ANGSTROEM)' in line:
+                for line  in log[(i + 2):]:
+                    if not line.strip():
+                        break
+                    if '---------------------------------' not in line :
+                        data = line.split()
+                        atoms.append(data[0])
+                        coords.append([float(c) for c in data[1:]])
+                        geometry_flag = True
+
+                if geometry_flag:
+                    break
+
+        # Assign appropriate mass to each atom in the molecule
+        for atom1 in atoms:
+            mass1, num1 = get_element_mass(atom1)
+            mass.append(mass1)
+            numbers.append(num1)
+        coord = numpy.array(coords, numpy.float64)
+        number = numpy.array(numbers, numpy.int)
+        mass = numpy.array(mass, numpy.float64)
+        if len(number) == 0 or len(coord) == 0 or len(mass) == 0:
+            raise LogError('Unable to read atoms from Orca geometry output file {0}'.format(self.path))
+
+        return coords, numbers, mass
+
+    def load_conformer(self, symmetry=None, spin_multiplicity=0, optical_isomers=None, label=''):
+        """
+        Load the molecular degree of freedom data from a log file created as
+        the result of a Orca "Freq" quantum chemistry calculation. As
+        CAUTION: The rotational entropy is not quite correctly treated here
+         because it includes a symmetry number that is not yet correctly
+         implemented in ORCA. Orca does not provide Rotational temperatures. Only E(rot) and E(trans) energies
+         Consider using another supported software (such as Gaussian or QChem) to calculate the frequencies and derive E0
+        """
+
+        raise NotImplementedError('Could not find a successfully load Orca scan:Parser is not inpemented')
```

Comment:
Could you rephrase this line and line 187?
There's a typo in `implemented`
let's make the format identical to that we have below, e.g. for `load_scan_pivot_atoms`:
`The load_scan_pivot_atoms method is not implemented for Orca Logs`